src/test/java/com/softlayer/api/http/HttpBasicAuthCredentialsTest.java

```diff
@@ -0,0 +1,23 @@
+package com.softlayer.api.http;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class HttpBasicAuthCredentialsTest {
+
+    @Test
+    public void testConstructor()
+    {
```

Comment:
Brace placement inconsistency here and below

Reply:
Bah! Need to add checkstyling to my editor to catch these...